[TW-864] Flows - add new FQL examples (#4541)
postmanlabs/postman-docs: src/pages/docs/postman-flows/flows-query-language/data-manipulation.md

``` json
"high-value customer"
```

## Partition an array

> Breaks an array up into smaller arrays and returns the smaller arrays as a list.

Reviewer: ```suggestion
The `$partition()` function breaks an array up into smaller arrays and returns the smaller arrays as a list.
```

### FQL

``` javascript
$partition(payments,2)
```

### Result

``` json
[
 [{"invoice_number": "101301","date": "2022-09-11T16:12:34.494Z","description": "recurring subscription","amount": 110.48},{"invoice_number": "101302","date": "2022-09-29T14:45:13.148Z","description": "one time purchase","amount": 24.49}],
 [{"invoice_number": "101303","date": "2022-10-11T16:12:34.683Z","description": "recurring subscription","amount": 110.48},{"invoice_number": "101304","date": "2022-10-12T11:45:22.182Z","description": "recurring subscription deluxe","amount": 35.56}]
]
```

## Perform an action on each value in an array

> 

Reviewer: ```suggestion
In this example, the `$map()` function gets the numerical values from the `amount` fields in the `payments` array and converts them to strings with the `$string()` function.  
```

### FQL

``` javascript
$map(input.payments[].amount,$string)
```

### Result

``` json
["110.48", "24.49", "110.48", "35.56"]
```

## Filter for values

> 

Reviewer: ```suggestion

in this example, the `$filter()` function returns values greater than 40 from `amount` fields in the `payments` array. 

```

### FQL

``` javascript
$partition(payments,2)
```

### Result

``` json
[
 [{"invoice_number": "101301","date": "2022-09-11T16:12:34.494Z","description": "recurring subscription","amount": 110.48},{"invoice_number": "101302","date": "2022-09-29T14:45:13.148Z","description": "one time purchase","amount": 24.49}],
 [{"invoice_number": "101303","date": "2022-10-11T16:12:34.683Z","description": "recurring subscription","amount": 110.48},{"invoice_number": "101304","date": "2022-10-12T11:45:22.182Z","description": "recurring subscription deluxe","amount": 35.56}]
]
```

## Perform an action on each value in an array

In this example, the `$map()` function gets the numerical values from the `amount` fields in the `payments` array and converts them to strings with the `$string()` function.

### FQL

``` javascript
$map(input.payments[].amount,$string)
```

### Result

``` json
["110.48", "24.49", "110.48", "35.56"]
```

## Filter for values

### FQL

``` javascript
$filter(input.payments[].amount,fn($v,$i,$a) { $v > 40})
```

### Result

``` json
[110.48, 110.48]
```

## Collapse an array to a single value

> Applies the function to every element in the array. In this case, it adds all the elements of the array together.

Reviewer: ```suggestion
The `reduce()` function applies a function to every element in the array. In this example, it adds all the elements of the array together.
```

### FQL

``` javascript
$reduce(input.payments[].amount,fn($i, $j){$i + $j})
```

### Result

``` json
281.01
```

## Sort an array

> Sorts an array by the function specified where `$j` is the current item and `$i` is the next item.

Reviewer: ```suggestion
The `$sort()` function sorts an array by the function specified where `$j` is the current item and `$i` is the next item.
```

### FQL

``` javascript
$sort(input.payments[].amount,fn($i, $j){$i < $j})
```

### Result

``` json
[110.48, 110.48, 35.56, 24.49]
```

## Perform an action on each value in an object

> 

Reviewer: ```suggestion

In this example, the `$each()` function converts every string value in an object to uppercase.

```
